packages/cli/src/push/abstract.push.ts

```diff
@@ -1,7 +1,7 @@
 import { EventEmitter } from 'events';
 import { assert, jsonStringify } from 'n8n-workflow';
 import type { IPushDataType } from '@/Interfaces';
-import { Logger } from '@/Logger';
+import type { Logger } from '@/Logger';
```

Comment:
These types are used by decorator metadata, and this change will most likely break automatic dependency injection.

Reply:
This was another eslint reported error after upgrading lint-related packages and it was auto fixed but in this case it's better to turn off this rule for `cli` (and maybe for `core`) as well